src/components/Button.js

```diff
@@ -1,16 +1,30 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-export default function Button({ name }) {
+export default function Button({ name, color, width }) {
+  let buttonClass = 'button';
+
+  if (color) {
+    buttonClass += ` ${color}`;
+  }
+
+  if (width) {
+    buttonClass += ` ${width}`;
+  }
+
   return (
-    <button type="button" className="button">{name}</button>
+    <button type="button" className={buttonClass}>{name}</button>
   );
 }
 
 Button.defaultProps = {
   name: '',
+  color: '',
+  width: '',
```

Comment:
the width property should actually be called wide, and it should be boolean ![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone3/WideProps.png)

Reply:
done